src/encoded/static/components/cart/add_multiple.js

```diff
@@ -86,7 +86,8 @@ class CartAddAllSearchComponent extends React.Component {
         const { savedCartObj, inProgress } = this.props;
         const cartName = (savedCartObj && Object.keys(savedCartObj).length > 0 ? savedCartObj.name : '');
         return (
-            <React.Fragment>
+            <div className="cart-add-all">
```

Comment:
Does "cart-add-all" follow the naming convention?

Reply:
Yes. “cart-add-all” represents a Block in BEM.